tests/ui/symbol-names/basic.legacy.stderr

```diff
@@ -1,10 +1,10 @@
-error: symbol-name(_ZN5basic4main17had874e876c8b1028E)
+error: symbol-name(_ZN5basic4main17h11bb905abadafc26E)
```

Comment:
Why do symbol hashes change?

Reply:
I guess (would need to try it out) that this is due to the addition of the `Safe` variant in the middle of the `Safety` enum that shifts the value of the `Default` variant.